newdle/client/src/components/NewdlesParticipating.js

```diff
@@ -98,11 +101,17 @@ function NewdleParticipation({
         <Icon name="clock outline" />
         <label>
           <em>
-            {finalDT
-              ? 'This newdle has finished'
-              : !slotsChosen.length
-              ? 'Awaiting your reply'
-              : `You replied with ${slotsChosen.length} timeslot(s)`}
+            {finalDT ? (
+              t`This newdle has finished`
+            ) : !slotsChosen.length ? (
+              t`Awaiting your reply`
+            ) : (
+              <Plural
+                value={slotsChosen.length}
+                one={`You replied with ${slotsChosen.length} timeslot`}
+                other={`You replied with ${slotsChosen.length} timeslots`}
```

Comment:
you can use `#` as a placeholder for the `value` here

Reply:
Addressed